src/main/java/org/assertj/core/api/Assertions.java

```diff
@@ -1139,12 +1139,48 @@ public static AtomicLongArrayAssert assertThat(AtomicLongArray actual) {
    *
    * @param shouldRaiseThrowable The lambda with the code that should raise the exception.
    * @return The captured exception or <code>null</code> if none was raised by the callable.
+   * @see #catchThrowableOfType(ThrowingCallable, Class)
    */
   public static Throwable catchThrowable(ThrowingCallable shouldRaiseThrowable) {
     return AssertionsForClassTypes.catchThrowable(shouldRaiseThrowable);
   }
 
   /**
+   * Allows catching an {@link Throwable} of a specific type more easily when used with Java 8 lambdas.
```

Comment:
Typo `an` -> `a` (fix it in `catchThrowable` as you go)

Reply:
Done.